src/Consolonia.Core/Drawing/PixelBufferImplementation/Symbol.cs

```diff
@@ -45,22 +68,49 @@ public Symbol(byte boxPattern)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public Symbol(string glyph)
+        public Symbol(string glyph, byte? width = null)
         {
-            if (glyph.Length == 1)
+            Pattern = 0;
+            Complex = null;
+            Character = char.MinValue;
+            if (string.IsNullOrEmpty(glyph))
```

Comment:
What is the case glyph is null? I feel it's misconception that null and empty are same. Why do we need null glyph at all?